sympy/printing/str.py

```diff
@@ -645,6 +645,9 @@ def _print_Uniform(self, expr):
     def _print_Union(self, expr):
         return ' U '.join(self._print(set) for set in expr.args)
 
+    def _print_SymmetricDifference(self, expr):
+        return ' ^ '.join(self._print(set) for set in expr.args)
```

Comment:
Use the `∆` unicode symbol for unicode printing. Also add latex printer, use can use `\triangle` for the symbol.